frontend/public/components/pod.tsx

```diff
@@ -405,11 +406,11 @@ export const PodsDetailsPage: React.FC<PodDetailsPageProps> = (props) => (
     menuActions={menuActions}
     pages={[
       {
-        href: 'dashboard', // TODO: make it default once additional Cards are implemented
+        href: POD_DETAIL_DASHBOARD_HREF,
```

Comment:
I'd suggest to stick with existing convention

```ts
href: '',
```

which makes it clear that this is the default landing page/tab within the details page.

Having a constant for an empty string adds one more level of indirection.

Reply:
done